install/migrations/update_10.0.x_to_11.0.0/commonitilobject_commonitilobject.php

```diff
@@ -64,7 +64,7 @@
         UNIQUE KEY `unicity` (`changes_id_1`,`changes_id_2`),
         KEY `changes_id_2` (`changes_id_2`)
         ) ENGINE=InnoDB DEFAULT CHARSET={$default_charset} COLLATE={$default_collation} ROW_FORMAT=DYNAMIC;";
-    $DB->doQueryOrDie($query, "10.1.0 add table glpi_changes_changes");
+    $DB->doQueryOrDie($query, "11.0.0 add table glpi_changes_changes");
```

Comment:
I guess we could avoid adding version in migration messages; version is already displayed while migrating (not a blocker)